moto/s3/responses.py

```diff
@@ -2259,6 +2259,7 @@ def _key_response_post(
                 multipart=multipart,
                 encryption=multipart.sse_encryption,
                 kms_key_id=multipart.kms_key_id,
+                request_method=request.method,
```

Comment:
This would send a `POST` event immediately after the `CompleteMultipartUpload`-action is finished. If I understand it correctly, AWS will only send a single event in this case. So we should avoid sending a `POST` event completely here.